include/tachimawari/control/sdk/module/dynamixel_sdk.hpp

```diff
@@ -52,10 +52,12 @@ class DynamixelSDK : public ControlManager
 
   void set_port(const std::string & port_name) override;
 
+
   bool connect() override;
   void disconnect() override;
 
   bool ping(uint8_t id) override;
+  bool ping_core(uint8_t id) override;
```

Comment:
What's the difference between `ping_core` and `ping` methods? I think it's the same and the old one is enough.

Reply:
updated